dev_guide/quota.adoc

```diff
@@ -130,3 +131,30 @@ resourcequotas          1       1
 services                3       5
 ----
 
+[[configuring_quota_sync_period]]
+
+== Configuring the Quota Synchronization Period
+
+When a set of resources are deleted, the synchronization timeframe of resources
+is determined by the `*resource-quota-sync-period*` setting in the
+`*/etc/openshift/master/master-config.yaml*` file. Before your quota usage is
+restored, you may encounter problems when attempting to reuse the resources.
+Change the `*resource-quota-sync-period:*` setting to have the set of resources
+regenerate at the desired amount of time (in seconds) and for the resources to
+be available again:
+
+====
+----
+kubernetesMasterConfig:
+  apiLevels:
+  - v1beta3
+  - v1
+  apiServerArguments: null
+  controllerArguments:
+    resource-quota-sync-period:
+      - "10s"
+----
+====
+
+Adjusting the regeneration time can be helpful for creating resources and
```

Comment:
Increasing the sync period places additional load on the openshift master, so any change should be made to balance overall system performance.